app/qml/MMMapThemePanel.qml

```diff
@@ -95,15 +95,55 @@ Drawer {
 
       delegate: delegateItem
 
-      TextHyperlink {
+      // No themes message + link
+      Rectangle {
         anchors.fill: parent
+        color: __style.whiteColor
         visible: parent.count === 0
-        text: qsTr("Project has no themes defined. See %1how to setup themes%2.")
-        .arg("<a href='"+ __inputHelp.howToSetupThemesLink +"'>")
-        .arg("</a>")
-        color: __style.nightColor
-        linkColor: __style.forestColor
-        font: __style.t3
+
+        Column {
+          width: parent.width
+          spacing: 20 * __dp
+          leftPadding: 20 * __dp
+          rightPadding: 20 * __dp
+          bottomPadding: 20 * __dp
+          anchors.horizontalCenter: parent.verticalCenter
+
+          Image {
+            source: __style.uploadImage
+            anchors.horizontalCenter: parent.horizontalCenter
+          }
+
+          Text {
+            text: qsTr("There are currently no map themes")
+            anchors.horizontalCenter: parent.horizontalCenter
+            font: __style.t1
+            width: parent.width - 2*20 * __dp
+            color: __style.forestColor
+            visible: text.length > 0
```

Comment:
this does not make sense, text is always defined as "There are currently no map themes"